src/main/java/core/basesyntax/model/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.model;
+
+public class RightTriangle extends Color {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * firstLeg * secondLeg;
+    }
+
+    @Override
+    public void draw() {
```

Comment:
The method 'draw' should not begin with an empty line. Please remove the empty line above the method declaration.